syft/pkg/license_set.go

```diff
@@ -1,13 +1,13 @@
+//nolint:dupl
 package pkg
 
 import (
 	"fmt"
 	"sort"
 
-	"github.com/mitchellh/hashstructure/v2"
-
 	"github.com/anchore/syft/internal/log"
 	"github.com/anchore/syft/syft/artifact"
+	"github.com/mitchellh/hashstructure/v2"
```

Comment:
Run `make lint-fix` to run the formatters; this will fix some of the lint issues

Reply:
done